src/ripple/app/main/Application.cpp

```diff
@@ -1143,11 +1142,11 @@ bool ApplicationImp::setup()
 
     getLedgerDB ().getSession ()
         << boost::str (boost::format ("PRAGMA cache_size=-%d;") %
-                        (config_->getSize (siLgrDBCache) * 1024));
+                        (config_->getSize (siLgrDBCache) * 1_kb));
```

Comment:
I don't think I can give a thumbs up for this until we can come up with a resolution for the type change problem. Consider the code above. If someone changes that to move the minus sign out of the format and into the calculation, we're computing the wrong value. We'll certainlly catch those problems when reviewing this changeset when we all have the issue in mind. However, it's going to be an ongoing maintence issue to remember that `_kb` and `_mb` are `unsigned long long`.